.github/workflows/prover-test.yaml

```diff
@@ -1,18 +1,22 @@
 name: "Prover Test"
 on:
-  # TODO enable this when we have time
-  # pull_request:
-  # push:
-  #   branches:
-  #     - main
-  #     - devnet
-  #     - testnet
-  #     - auto
-  #     - canary
-
+  pull_request:
+    paths:
+      - '**.move'
+      - 'aptos-move/move-deps/Cargo.toml'
+  push:
+    branches:
+      - main
+      - devnet
+      - testnet
+      - auto
+      - canary
+    paths:
+      - '**.move'
+      - 'aptos-move/move-deps/Cargo.toml'
   # Allow triggering manually
   workflow_dispatch:
-  # Until enabled on PR, run continuously
+  # Until enabled on all PRs, run continuously
```

Comment:
maybe change this to hourly? (* instead of */3)

Reply:
Done. Thanks!